src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -38,12 +38,6 @@ internal partial class SymbolSearchUpdateEngine
         public const string HostId = "RoslynNuGetSearch";
         private const string MicrosoftAssemblyReferencesName = "MicrosoftAssemblyReferences";
 
-        /// <summary>
-        /// Cancellation support for the task we use to keep the local database up to date.
-        /// When VS shuts down it will dispose us.  We'll cancel the task at that point.
-        /// </summary>
-        private readonly CancellationToken _updateCancellationToken;
```

Comment:
Or is this ok because you're not tying this to a *feature*'s cancellation token, but are instead tying it to some long lived cancellation token that is only triggered when VS shuts down?